.eslintrc.js

```diff
@@ -64,5 +67,26 @@ module.exports = {
         }],
         "simple-import-sort/imports": "error",
         "simple-import-sort/exports": "off",
+        "header/header": [2, "block", [
+            "-",
+            " *",
+            " * Hedera JSON RPC Relay",
+            " *",
+            {"pattern": "^\\s\\*\\sCopyright \\(C\\) 20(1[8-9]|[2-9]\\d)(?:-20(1[8-9]|[2-9]\\d))? Hedera Hashgraph, LLC$", "template": ` * Copyright (C) ${new Date().getFullYear()} Hedera Hashgraph, LLC`},
```

Comment:
Just making sure, this will manage the year ranges also e.g. 2022 - 2024 for older file.

Reply:
Yes, it manages year ranges and single year